source/tutorials/kubernetes.rst

```diff
@@ -180,71 +350,143 @@ Configure and run CRI-O + kata-runtime
+
+What is Kata Containers?
+========================
+
+`Kata Containers`_ is a secure container runtime leveraging lightweight
+virtual machines that feel and perform like containers, but provide stronger
+workload isolation using hardware virtualization technology as a second layer
+of defense for untrusted workloads or multi-tenant scenarios.
+
+Kata Containers can be allocated on a per-pod basis, so you can mix and match
+both on the same host to suit your needs.
+
+The Kata Containers\* (kata-runtime) adheres to :abbr:`OCI (Open Container
+Initiative*)` guidelines and works seamlessly with Kubernetes through CRI+O.
+
+cloud-native-setup automation
+=============================
+
+Instead of manually installing a Kubernetes cluster as described in this
+tutorial, you can clone the `cloud-native-setup repository`_  on your system
+and follow the instructions. 
+
+It includes helper scripts to automate configuration.
+
+
+Related topics
+==============
+
+* `Understanding basic Kubernetes architecture`_
 
 * Installing a `pod network add-on`_
 
 * `Joining your nodes`_
 
-Cloud native setup automation
-*****************************
+* `Deploying an application to your cluster`_
+
+*  See our document on :ref:`Kubernetes best practices <kubernetes-bp>`
 
-Optional: Clone the `cloud-native-setup`_ repository on your system and follow the instructions. This repository includes helper scripts to automate configuration.
+
+Troubleshooting
+***************
 
 Package configuration customization (optional)
-**********************************************
+==============================================
 
-|CL| is a stateless system that looks for user-defined package configuration files in the :file:`/etc/<package-name>` directory to be used as default. If user-defined files are not found, |CL| uses the distribution-provided configuration files for each package.
+|CL| is a stateless system that looks for user-defined package configuration
+files in the :file:`/etc/<package-name>` directory to be used as default. If
+user-defined files are not found, |CL| uses the distribution-provided
+configuration files for each package.
 
-If you customize any of the default package configuration files, you **must** store the customized files in the :file:`/etc/` directory. If you edit any of the distribution-provided default files, your changes will be lost in the next system update as the default files will be overwritten with the updated files.
+If you customize any of the default package configuration files, you **must**
+store the customized files in the :file:`/etc/` directory. If you edit any of
+the distribution-provided default files, your changes will be lost in the next
+system update as the default files will be overwritten with the updated files.
 
-For example, to customize CRI-O configuration in your system, run the following commands:
+For example, to customize CRI-O configuration in your system, run the
+following commands:
```

Comment:
This code block explanation seems redundant.  When you enable the crio service it automatically creates /etc/crio/crio.conf.  You don't have to copy it.

Reply:
I agree, this was existing content.